driver/cmd/ibm-spectrum-scale-csi/main.go

```diff
@@ -143,14 +147,22 @@ func setContext() context.Context {
 	return ctx
 }
 
-func getLogEnv() string {
+func getLogEnv() (string, string) {
 	level := os.Getenv(logLevel)
-	return level
+	logLevel := strings.ToUpper(level)
+	if logLevel != TRACE.String() || logLevel != DEBUG.String() || logLevel != INFO.String() || logLevel != WARNING.String() || logLevel != ERROR.String() || logLevel != FATAL.String() {
```

Comment:
The logic doesn't seem right here, e.g. if env var has value "FATAL" --> logLevel is still going to be set as "INFO". The correct set of conditions would be either
```
if !(logLevel == "TRACE" || logLevel == "DEBUG" || ...)
```
or
```
if logLevel != "TRACE" && logLevel != "DEBUG" &&
```

Reply:
Modified condition to the first suggestion and tested as well... second will not work

Reply:
Thanks for correcting the conditions! 2nd suggestion is another form of the 1st one, so it works

Reply:
yeah correct I got confused it would work :)